packages/expo-store-review/ios/StoreReviewModule.swift

```diff
@@ -0,0 +1,26 @@
+import ExpoModulesCore
+import StoreKit
+
+public class StoreReviewModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoStoreReview")
+
+    AsyncFunction("isAvailableAsync") { () -> Bool in
+      return true
+    }
+
+    AsyncFunction("requestReview") { (promise: Promise) -> Void in
```

Comment:
Since the function's body is synchronous, you can remove the promise from arguments and then just return the value that the JS promise should resolve to

```suggestion
    AsyncFunction("requestReview") {
```